### PR TITLE
feat: add all needfull files for work

### PR DESCRIPTION
**Pull request by MaxMaroz**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    bio = db.Column(db.Text)
+
+    def __repr__(self):
+        return f'<Student {self.firstname}>'
+
+
+@app.route('/')
+def index():
+    students = Student.query.all()
+    return render_template('index.html', students=students)
```
Comment (arty-ms) [inline app.py]: There is a route '/' but there is no corresponding index.html for this route

File: app.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] =\
+        'sqlite:///' + os.path.join(basedir, 'database.db')
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+
+db = SQLAlchemy(app)
+
```
Comment (arty-ms) [inline app.py:16]: please remove one tab

File: app.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+@app.route('/<int:student_id>/')
+def student(student_id):
+    student = Student.query.get_or_404(student_id)
+    return render_template('student.html', student=student)
+
+
+
+@app.route('/create/', methods=('GET', 'POST'))
```
Comment (arty-ms) [inline app.py:37]: just make it as 'POST' and you will not need to check `if request.method == 'POST':`

File: app.py [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+        bio = request.form['bio']
+        student = Student(firstname=firstname,
+                          lastname=lastname,
+                          email=email,
+                          age=age,
+                          bio=bio)
+        db.session.add(student)
+        db.session.commit()
+
+        return redirect(url_for('index'))
```
Comment (arty-ms) [inline app.py]: redirecting to no-where)

File: templates/base.html [hunk 5]
```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>{% block title %} {% endblock %} - FlaskApp</title>
+    <style>
```
Comment (arty-ms) [inline templates/base.html]: please, move styles to separate file, styles.css, and connect html to it

File: app.py [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+db = SQLAlchemy(app)
+
+
+class Student(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    firstname = db.Column(db.String(100), nullable=False)
+    lastname = db.Column(db.String(100), nullable=False)
+    email = db.Column(db.String(80), unique=True, nullable=False)
+    age = db.Column(db.Integer)
+    created_at = db.Column(db.DateTime(timezone=True),
```
Comment (arty-ms) [inline app.py]:
```
created_at = db. Column(
    db.DateTime(timezone=True),
    server_default=func.now()
)
```

File: app.py [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+                           server_default=func.now())
+    bio = db.Column(db.Text)
+
+    def __repr__(self):
+        return f'<Student {self.firstname}>'
+
+
```
Comment (arty-ms) [inline app.py]: remove unnecessary empty line
Just one line needed

File: app.py [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+    students = Student.query.all()
+    return render_template('index.html', students=students)
+
+
+@app.route('/<int:student_id>/')
+def student(student_id):
+    student = Student.query.get_or_404(student_id)
+    return render_template('student.html', student=student)
+
+
```
Comment (arty-ms) [inline app.py]: remove unnecessary 2 empty lines

**Comment by MaxMaroz** [comment]
Correct some mistakes

**Review by arty-ms — APPROVED** [review]
You can merge the PR
LGTM!

**Pull request merged**